cmd/query/app/grpc_handler.go

```diff
@@ -46,36 +52,36 @@ func NewGRPCHandler(queryService *querysvc.QueryService, logger *zap.Logger, tra
 	return gH
 }
 
-// GetTrace is the GRPC handler to fetch traces based on trace-id.
+// GetTrace is the gRPC handler to fetch traces based on trace-id.
 func (g *GRPCHandler) GetTrace(r *api_v2.GetTraceRequest, stream api_v2.QueryService_GetTraceServer) error {
 	trace, err := g.queryService.GetTrace(stream.Context(), r.TraceID)
 	if err == spanstore.ErrTraceNotFound {
-		g.logger.Error("trace not found", zap.Error(err))
-		return err
+		g.logger.Error(msgTraceNotFound, zap.Error(err))
+		return status.Errorf(codes.NotFound, "%s: %v", msgTraceNotFound, err)
 	}
 	if err != nil {
-		g.logger.Error("Could not fetch spans from backend", zap.Error(err))
-		return err
+		g.logger.Error("failed to fetch spans from the backend", zap.Error(err))
```

Comment:
Logs may be capitalized and with periods while error strings need not (because they are pulled into stack traces)

Reply:
I know, but I felt it's simpler to keep strings consistent between errors and logs